modular_skyrat/modules/interaction_menu/code/interaction_datum.dm

```diff
@@ -121,10 +121,10 @@ GLOBAL_LIST_EMPTY_TYPED(interaction_instances, /datum/interaction)
 	if(lewd)
 		user.adjustPleasure(user_pleasure)
 		user.adjustArousal(user_arousal)
-		user.adjustPain(user_pain)
+		user.adjust_pain(user_pain)
 		target.adjustPleasure(target_pleasure)
 		target.adjustArousal(target_arousal)
```

Comment:
`adjust_pleasure()` and `adjust_arousal()` should also be the names of those two other procs, sorry if that wasn't clear in my last review.